package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "cross-env NODE_PATH=./styleguide concurrently --kill-others \"nodemon --watch . --ext js,jsx server\" \"node support/webpack.server.js\" \"gulp css:watch\"",
     "lint": "eslint . --ext .jsx --ext .js",
-    "test": "cross-env NODE_ENV=test cross-env NODE_PATH=./styleguide mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
+    "test": "jest",
```

Comment:
Excellent.  I like jest better.  I just hadn't gotten around to changing the test suite yet.  Thanks for doing this!